b4model/feature_splitter.py

```diff
@@ -0,0 +1,22 @@
+import pandas as pd
+
+def feature_splitter(x):
+    """ Splits dataset column names into a tupple of categorical and numerical lists
+
+    Parameters
+    ----------
+    x : DateFrame
+
+    Returns
+    -------
+    tupple: 
```

Comment:
- small grammatical error in 'tupple', should be spelled 'tuple'
- otherwise, looks good!